### ISSUE TITLE
Do you have Python code?

### ISSUE THREAD

**Issue by zhouhao-learning** [yaojieliu/ICCVW2017-DenseFaceAlignment#1]
Hello, I see that your code is implemented in matlab. Do you have Python code? Thank you.

**Comment by yaojieliu** [comment]
No, you can use matlab library compiler and matlab runtime to compile it to a python library. You are more than welcome to contribute.
